runtime/cpp/cmake/SourcesAndHeaders.cmake

```diff
@@ -23,5 +23,6 @@ set(skel_includes
 )
 
 set(test_sources
-    src/bpf_api_test.cpp
+    # src/bpf_api_test.cpp
```

Comment:
Why comment out the src/bpf_api_test.cpp?

Reply:
Oh i forget to remove the comment..

Reply:
Fixed now